proposed/2021/BulkRestoreCoordination.md

```diff
@@ -127,11 +138,33 @@ The proposal is that each time NuGet determines a restore might need run, it che
     }
 ```
 
+#### Implementation details
+
+- Given that there are risks with this proposal, the NuGet implementation will have a means of disabling this feature.

```

Comment:
To be clear, disabling this feature means returning to current (released) behavior?  That means the 10-second sliding window can't be removed; it must be disabled when this feature is enabled and enabled otherwise.

Reply:
We'd go back to an old implementation that will likely have a longer sliding window at solution load.